### ISSUE TITLE
Public model-drift leaderboard (hallucination over deep multi-agent chains) — preregistered

### ISSUE THREAD

**Issue by alizahidraja** [alizahidraja/isnad#71]
## Context (verified against repo)

Two related-but-distinct artifacts already exist:

- **ISNAD-Bench** (#58, shipped in 2.6.0) — validates grading against *classical hadith* ground truth (κ = 0.87).
- **`experiments/adversarial_benchmark/`** (#50) — measures ISNAD's own corruption-detection.

Neither is a **model** leaderboard.

## Proposal

An open, independent leaderboard (e.g. Hugging Face Spaces) measuring how popular models (GPT-4o, Claude, Llama-3, …) drift on factual claims over deep multi-agent chains, with ISNAD lineage attached — i.e. "how much does a claim decay as it passes through N agents?"

## Honesty constraint (critical — this is where the original pitch over-claims)

- **Do not** frame this as "proving ISNAD catches hallucination decay faster than standard RAG". That is a *hypothesis to test*, not a fact to assert. If the experiment shows it, great — publish the number; if not, publish the negative result.
- A "factual drift" leaderboard needs a **preregistered ground-truth source** and must state exactly what it measures (and what it can't), per the repo's honesty charter in `bench/README.md`.

This is a research/placement artifact, not core library code.

**Comment by samarthputhraya** [comment]
On the preregistration point — one axis worth pinning alongside the ground-truth source is *which endpoint, on what date*. A board over hosted model names is measuring a moving target, and the movement isn't hypothetical:

```
$ curl -s -o /dev/null -w '%{http_code}\n' https://integrate.api.nvidia.com/v1/models/meta/llama-3.1-8b-instruct
410
```

The body gives an EOL of `2026-08-26T09:00:00Z`, three days ago, and the id is gone from that host's `/v1/models` listing. So a "Llama-3" row published without a host and a date stops being reproducible fairly quickly — and a *drift* leaderboard is the one place where that matters twice, because movement in the x-axis and movement in the thing being measured look the same in the published table.

Are you planning to pin rows to a provider + dated snapshot, or to re-run the whole board on a cadence and publish it as a time series? The second seems like it would make the decay-over-N-agents question harder to read, but it's the only one that survives ids retiring under you.

**Comment by alizahidraja** [comment]
Panel re-audit (2026-08-29, market+scope+vision lanes): status unchanged — correctly deferred until real users exist. See issue #130 for the ranked rationale. Key finding: #54's undetectable half (correlated training data across distinct model families) is active 2026 research (arXiv 2604.07650 'behavioral entanglement') and nobody productizes it — the most on-moat of the five, but research-stage with no buyer yet.
